### ISSUE TITLE
Add more relevant information in the 'compilers' section

### ISSUE THREAD

**Issue by ForgottenProgramme** [conda-forge/conda-forge.github.io#1357]
https://conda-forge.org/docs/maintainer/knowledge_base.html#compilers

Currently, the 'Compilers' section is incomplete and unhelpful.

Refer: https://github.com/conda-forge/conda-forge.github.io/issues/893#issuecomment-634881487

The link mentioned in the comment by @isuruf doesn't exist as conda-forge doesn't use wiki anymore.
(https://github.com/conda-forge/conda-forge.github.io/issues/893#issuecomment-634915638)

**Suggested Solution:** 

The following list should be added to the compiler section: 

**Activation recipes**
1.GCC - https://github.com/conda-forge/ctng-compiler-activation-feedstock
2. Clang, OSX - https://github.com/conda-forge/clang-compiler-activation-feedstock
3. gfortran, OSX - https://github.com/conda-forge/gfortran_osx-64-feedstock

**Implementation packages**
1. GCC - https://github.com/conda-forge/ctng-compilers-feedstock
2. gfortran, OSX - https://github.com/conda-forge/gfortran_impl_osx-64-feedstock
3. clang
https://github.com/conda-forge/llvmdev-feedstock
https://github.com/conda-forge/libcxx-feedstock
https://github.com/conda-forge/clangdev-feedstock
https://github.com/conda-forge/openmp-feedstock
https://github.com/conda-forge/tapi-feedstock

4. OSX - https://github.com/conda-forge/cctools-and-ld64-feedstock

**Comment by ForgottenProgramme** [comment]
Suggestions invited. 

@viniciusdc , I'm taking this up.

**Comment by jakirkham** [comment]
The wiki was disabled, which is why the link didn't work. Reenabled it so we can pull that information. Though I think we are trying to not use the wiki in the future. So we may want to disable that again after this is done

**Comment by jakirkham** [comment]
Windows also reuses Visual Studio with some activation scripts

Also we have a CUDA compiler for GPU code

I believe there are Rust and Go compilers as well. Not sure if there are any other languages I'm missing

**Comment by ForgottenProgramme** [comment]
Alright, 
so we add information about the CUDA compilers and then check for Rust and Go.

**Comment by viniciusdc** [comment]
> The wiki was disabled, which is why the link didn't work. Reenabled it so we can pull that information. Though I think we are trying to not use the wiki in the future. So we may want to disable that again after this is done

yup, @jakirkham  I copied the info there and passed it to @ForgottenProgramme as a source for this issue body. I will close the wiki again.

**Comment by ForgottenProgramme** [comment]
@viniciusdc , @jakirkham 
Where should this list be added though?

**Comment by ForgottenProgramme** [comment]
Also, Is this is the feedstock repo for CUDA : https://github.com/conda-forge/cudatoolkit-feedstock ?

**Comment by viniciusdc** [comment]
@jakirkham Maybe we could use https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml to auto update this information in the future, what do you think?

**Comment by ForgottenProgramme** [comment]
@viniciusdc ,
I was thinking we could update the list of supported compilers.

**Comment by jakirkham** [comment]
> @jakirkham Maybe we could use https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml to auto update this information in the future, what do you think?

Maybe. @CJ-Wright @beckermr @isuruf  do you have thoughts on this?

**Comment by hitblunders** [comment]
@ForgottenProgramme are you working on this?
